examples/persistence_service/persistent_storage/c++11/hello_world_subscriber.cxx

```diff
@@ -93,15 +90,16 @@ int main(int argc, char *argv[])
             domain_id = atoi(argv[i++]);
         } else if (param == "-sample_count" && i < argc) {
             sample_count = atoi(argv[i++]);
-        } else if (param == "-drs" && i < argc) {
-            drs = (atoi(argv[i++]) == 1);
+        } else if (param == "-persistent" && i < argc) {
+            is_persistent = atoi(argv[i++]);
```

Comment:
```suggestion
            is_persistent = strcmp(atoi(argv[i++], 1) == 0 ? true : false);
```

Reply:
Are we robust to null arguments?

Reply:
Yes .. if an option is used .. it needs to be followed by its value. We are also protected from garbage values by `atoi` itself - https://cplusplus.com/reference/cstdlib/atoi/.